shared/src/types/admin/Contribution.ts

```diff
@@ -20,7 +20,7 @@ export enum StatusKey {
 
 export type Contribution = {
 	source: ContributionSourceKey;
-	created: Date | Timestamp; // This is a hack. Firestore returns a Timestamp, but Firecms seems to expect a Date for a date picker. Will look into this.
```

Comment:
As far as I can tell, the proper typing would be `Timestamp` here. I added a `@ts-ignore` comment to the line in the admin project that complains about this not being a `Date`.

Reply:
In the past, the problem was that the firecms admin required the Date as input, but firestore actually returns a Timestamp. Are you sure everything still works on the admin side? Would be nice to have it only as Timestamp 👍